addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -1134,9 +1134,12 @@ function listGroupToTable(editable) {
 function normalizeColors($editable) {
     const editable = $editable.get(0);
     for (const node of editable.querySelectorAll('[style*="rgb"]')) {
-        const rgbMatch = node.getAttribute('style').match(/rgb?\(([\d\.]*,?\s?){3,4}\)/g);
+        const rgbMatch = node.getAttribute('style').match(/rgb[a]*?\(([\d\.]*,?\s?){3,4}\)/g);
```

Comment:
Shouldn't this variable be renamed to `rgbaMatch`, then ?

Reply:
Yes totally right.